HLTrigger/HLTfilters/test/triggerResultsFilter_producer.py

```diff
@@ -2,6 +2,14 @@
 
 process = cms.Process('HLT')
 
+process.load('FWCore.MessageService.MessageLogger_cfi')
+#process.MessageLogger.cerr.INFO = cms.untracked.PSet(
+#    reportEvery = cms.untracked.int32(1), # every!
+#    limit = cms.untracked.int32(-1)       # no limit!
+#    )
+#process.MessageLogger.cerr.FwkReport.reportEvery = 100 # only report every 100th event start
+#process.MessageLogger.cerr_stats.threshold = 'INFO' # also info in statistics
```

Comment:
why these commented-out lines ?

Reply:
In short, convenience. I used L10 to limit printouts while testing, and I thought it might be convenient to leave these lines there similarly to what is in one of the other configs, [triggerResultsFilter_by_TriggerResults.py#L13](https://github.com/cms-sw/cmssw/blob/7d2e3ad9f9384ce8698467f30e64d7732b021656/HLTrigger/HLTfilters/test/triggerResultsFilter_by_TriggerResults.py#L13).

I can remove them if you prefer, of course.

Reply:
Quite the opposite... if they are useful for testing, you could also uncomment them, since this is a test file.
Would there be any downsides ?

Reply:
I don't think so, let me have a look and update.

Reply:
We could use the snippet below in the 3 configs for the unit-tests. What do you think?

With this, the event number is reported only 10 times instead of 1000, warnings/errors and the `TrigReport` messages are reported. The commented-out lines enable all info-level outputs for every event, and are left there for debugging.
```py
process.options.wantSummary = True

process.load('FWCore.MessageService.MessageLogger_cfi')
process.MessageLogger.cerr.FwkReport.reportEvery = 100 # only report every 100th event start
process.MessageLogger.cerr.enableStatistics = False # enable "MessageLogger Summary" message
process.MessageLogger.cerr.threshold = 'INFO' # change to 'WARNING' not to show INFO-level messages
## enable reporting of INFO-level messages (default is limit=0, i.e. no messages reported)
#process.MessageLogger.cerr.INFO = cms.untracked.PSet(
#    reportEvery = cms.untracked.int32(1), # every event!
#    limit = cms.untracked.int32(-1)       # no limit!
#)
```